src/components/CreateComment.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react"
+import { postComment } from "../utils/utils"
+import { Link } from "react-router-dom"
+
+function CreateComment({ user, review_id, count, setCount }) {
+    const [isPosted, setIsPosted] = useState(true)
+    const [value, setValue]=useState('')
+    if (user === '0') {
+        return <h3><Link to="/login">login</Link> to post a comment</h3>
+    }
+    function handleSubmit(event) {
+        event.preventDefault()
+        setIsPosted(false)
+        if (value !== '') {
+            postComment(review_id, user, value)
+                .then((data) => {
+                    setValue('')
+                    setIsPosted(true)
+                    setCount(count + 1)
+                })
+        }
+    }
+    function handleChange(event) {
+        setValue(event.target.value)
+    }
+    if (!isPosted) {
+        return <p>posting: {value}</p>
```

Comment:
Currently, it is saying Posting: when I try and post an empty comment and it just hangs there. There is no error so it looks like the if statement on line 14 is working. Maybe the if statement on line 26 can be moved into the handleSubmit function?